release_notes.md

```diff
@@ -0,0 +1 @@
+* New feature: Unless specified otherwise in app settings, the PowerShell worker will set the AZUREPS_CHECK_FOR_UPGRADE environment variable to "False" in all runspaces to opt-out of the version upgrade notification in Azure PowerShell added by that team. 
```

Comment:
Please remove this file from your change. For the PowerShell language worker, we hand craft the release notes right before a worker release. We an issue opened to remove the `release_notes.md` file from the repo. 

/cc @davidmrdavid

Reply:
done